File: cpp/raindrops/raindrops.cpp
#include <string>
#include "raindrops.h"

using namespace std;

namespace raindrops {

    string convert(int input){
        string sortie = "";
        if (input%3 == 0){
            sortie += "Pling";
        }
        if (input%5 == 0){
            sortie += "Plang";
        }
        if (input%7 == 0){
            sortie += "Plong";
        }
        if (input%3 != 0 && input%5 != 0 && input%7 != 0){
            sortie += to_string(input);
        }
        
        return sortie;
    }

}  // namespace raindrops

File: python/raindrops/raindrops.py
def convert(number):
    resultat = ""
    
    if number%3 == 0:
        resultat += "Pling"
    if number%5 == 0:
        resultat += "Plang"
    if number%7 == 0:
        resultat += "Plong"
    if number%3 != 0 and number%5 != 0 and number%7 != 0:
        resultat += str(number)
        
    return resultat
